dubbo-cluster/src/main/java/com/alibaba/dubbo/rpc/cluster/router/script/ScriptRouter.java

```diff
@@ -114,13 +106,4 @@ public URL getUrl() {
         }
     }
 
-    @Override
-    public int compareTo(Router o) {
-        if (o == null || o.getClass() != ScriptRouter.class) {
-            return 1;
-        }
-        ScriptRouter c = (ScriptRouter) o;
-        return this.priority == c.priority ? rule.compareTo(c.rule) : (this.priority > c.priority ? 1 : -1);
-    }
-
 }

```

Comment:
why do you remove compareTo method ? It is different from super class

Reply:
compareTo is centralized management by the `AbstractRouter`，make the behavior the same as 2.7.x. Only override `getPriority` to control the priority.